src/main/java/org/mskcc/cbio/oncokb/web/rest/slack/DropdownEmailOption.java

```diff
@@ -149,6 +159,11 @@ public DropdownEmailOptionBuilder mailType(MailType mailType) {
             return this;
         }
 
+        public DropdownEmailOptionBuilder isNotModalEmail() {
```

Comment:
some emails (only trial atm) do not require view modal, it used to throw an error as it expected view modal parameter but new logic is fine.